Revise Transparency Note for CLI Agent for AKS (#11)
Azure/agentic-cli-for-aks: TRANSPARENCY-NOTE.md

The CLI Agent for AKS is being evaluated through a combination of internal testing and  programmatic evaluations (evals) designed to ensure its diagnostic capabilities are accurate, relevant, and impactful. For Programmatic evals, we measured standard responsible AI metrics such as Groundedness, UPIA and XPIA Jailbreak, and Harmful Content, Conversation Quality (coherence, fluency etc.).  Internal bug bashes  and red-teams are conducted to rigorously test the agent’s behavior across a variety of cases including but not limited to - common issues such as node health degradation, DNS failures, upgrade disruptions, and pod scheduling problems., helping the team identify gaps in reasoning, tooling integration, and prompt execution.  A core metric for success is the accuracy of the agent’s diagnosis and the relevance of its recommendations—whether it correctly identifies the root cause and suggests actionable, context-aware mitigations.  We realize the dynamic nature of agentic-AI interactions, and we welcome user feedback as part of the preview, users can share feedback directly with us at - aksagentcli@service.microsoft.com or create Github issues here. 

> ## What are the limitations of [system or product name]? How can users minimize the impact of [system or product name]’s limitations when using the system?

Reviewer: ```suggestion
## What are the limitations of CLI Agent for AKS? How can users minimize the impact of  the CLI Agent’s limitations when using the system?
```
